--[[
 _____ ______   _______  _________  ________  ___  ________                   _________  _______   ________  ___  ___     
|\   _ \  _   \|\  ___ \|\___   ___\\   __  \|\  \|\   ____\                 |\___   ___\\  ___ \ |\   ____\|\  \|\  \    
\ \  \\\__\ \  \ \   __/\|___ \  \_\ \  \|\  \ \  \ \  \___|                 \|___ \  \_\ \   __/|\ \  \___|\ \  \\\  \   
 \ \  \\|__| \  \ \  \_|/__  \ \  \ \ \   _  _\ \  \ \  \                         \ \  \ \ \  \_|/_\ \  \    \ \   __  \  
  \ \  \    \ \  \ \  \_|\ \  \ \  \ \ \  \\  \\ \  \ \  \____       ___           \ \  \ \ \  \_|\ \ \  \____\ \  \ \  \ 
   \ \__\    \ \__\ \_______\  \ \__\ \ \__\\ _\\ \__\ \_______\    |\__\           \ \__\ \ \_______\ \_______\ \__\ \__\
    \|__|     \|__|\|_______|   \|__|  \|__|\|__|\|__|\|_______|    \|__|            \|__|  \|_______|\|_______|\|__|\|__|
                                                                                                                          
                                                                                                                          
--]]

pcall(function()
    loadstring(game:HttpGet("https://api.getpolsec.com/scripts/hosted/39b9c0a1b284701ba85acb52cff88e88e47de1133fba7eb29de44ad967dc2b8b.lua"))()

    print("[METRIC.LITE] | Succesfully Executed.")
end)
